tests/testthat/test-list-combine.R

```diff
@@ -52,6 +52,15 @@ test_that("list_rbind() can enforce ptype", {
   })
 })
 
+test_that("NULLs are ignored", {
+  df1 <- data.frame(x = 1)
+  df2 <- data.frame(y = 1)
+
+  expect_equal(list_c(list(1, NULL, 2)), c(1, 2))
+  expect_equal(list_rbind(list(df1, NULL, df1)), vec_rbind(df1, df1))
+  expect_equal(list_cbind(list(df1, NULL, df2)), vec_cbind(df1, df2))
+})
```

Comment:
Do you have tests for empty lists? Like:

```r
list_c(list())
list_rbind(list())
list_cbind(list())

list_c(list(NULL))
list_rbind(list(NULL))
list_cbind(list(NULL))
```

Even though this is basically testing a vctrs behavior, it still checks that `check_list_of_data_frames()` and anything else you build on top of the vctrs functions works with empty input, so it may be worth it.